src/event.js

```diff
@@ -424,8 +426,11 @@ jQuery.event = {
 			delegateCount = handlers.delegateCount,
 			cur = event.target;
 
+		// Support (at least): Chrome, IE9
```

Comment:
> f we don't know the fix version because that is also too far back in the past, then I'd support either fudging it or extending the grammar to accept Support: Chrome <=34 - <39 only.

I think if the fix is too far back in the past, the support comment would have been dropped already.